python/cog/server/worker.py

```diff
@@ -52,7 +59,7 @@ def __init__(self, predictor_ref: str, tee_output: bool = True) -> None:
         self._child = _ChildWorker(predictor_ref, child_events, tee_output)
         self._terminating = False
 
-    def setup(self) -> Iterable[_PublicEventType]:
+    def setup(self) -> Iterator[_PublicEventType]:
```

Comment:
Oops! This was probably my mistake.

Reply:
I think Iterable isn't wrong I just needed Iterator for the generator and didn't feel like importing both